### ISSUE TITLE
FluxMusicGUI

### ISSUE THREAD

**Issue by curtified** [feizc/FluxMusic#9]
I created a GUI out of your .ipynb file here:

[https://github.com/curtified/FluxMusicGUI](url)

**Comment by rmcc3** [comment]
Code has issues, but you have issues disabled on your repo. Very bad practice for open source.

**Comment by curtified** [comment]
Yeah I’m working on it. The cool thing about open source is you can clone the repo and fix them yourself

**Comment by curtified** [comment]
trying to get it to run on huggingface with CPU infer. feel free to tweak my code plz.

https://huggingface.co/spaces/flosstradamus/FluxMusicGUI
